addons/fluentbit/fluentbit.yaml

```diff
@@ -269,3 +269,6 @@ spec:
               Time_Keep On
               Time_Key requestReceivedTimestamp
               Time_Format %Y-%m-%dT%H:%M:%S.%L
+
```

Comment:
I think fluentbit revision needs to be bumped

Reply:
Oops, think I forgot that when I resolved conflicts w/ master. Fixed with f5b85a8f50bdfcc57de54c7407bcd52cac51b15d, thanks @joejulian.